packages/plugins/geth/src/index.js

```diff
@@ -50,6 +51,19 @@ class Geth {
         cb();
       }
     });
+
+    this.setupDevTxs();
+  }
+  
+  async setupDevTxs() {
+    const devTxs = new DevTxs(this.embark);
+    await devTxs.init();
+    try {
+      await devTxs.startRegularTxs();
+      this.logger.info("Regular transactions started");
```

Comment:
Would probably be better to change this message to make clearer that it's for Geth and because of stuck transaction. Also, when we are not in dev mode, I know it doesn't start, but we still show the message, which is not wanted.